### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ConnectToAzureStorage_ScreenReader: Screen reader not announces collapsed/expanded state for ""Azure Environment" combo box.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2299]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #MAS4.2.1; #VoiceOver; #Mac; #CELAAssessID_CosmosDB-FY20-Q2; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.0
MAC Version: Catalina 10.15 

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. "Connect to Azure Storage" popup opens.
4. Select "Add an Azure Account" radio button.
5. Start VoiceOver and using Tab key navigate to "Azure Environment" combo box and hit enter.
6. Observe Screen reader not announces the Expanded/Collapsed state.

**​Actual:​**
For first time only Voiceover announces, "Azure, Azure Environment: collapsed, button".
Once the combo box is expanded, Voiceover does not announce the expanded/collapsed state of 'Azure environments:' combo box. 
For eg. Voiceover says, "5 menu items, Azure".

**Same issue is observed in below Scenario:**
1. In "Connect with Name and Key" dialog, "Storage Domain" drop down
(Repro Steps: "Connect to Azure Storage" dialog -> select "Use a storage account name and key" radio button->Click Next).
2. In "Attach with Azure AD" dialog, "Tenant" drop down
(Repro Steps:"Connect to Azure Storage" dialog -> select "Add a resource via Azure Active Directory (Azure AD)->Click Next).

**Expected:**
Voiceover should properly announce the expanded/collapsed state for 'Azure environments:' combo box when it is expanded/ collapsed.

**UserImpact:**
Screen reader users may not be able to know the state of the combo box whether it is expanded/collapsed.

**Comment by MRayermannMSFT** [comment]
Looks like newer electrons work better with VoiceOver with regards to dropdowns.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT 
We will verify this issue Once "Microsoft Tools Storage Explorer" application is updated to Newer Verion(1.12.0).

**Comment by Alexandria-R** [comment]
Approved for exclusion as low impact.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
Verified the bug in the environment: 
Microsoft Azure Storage Explore
Version: 1.29.2
AzCopy Version: 10.18.1

Bug is fixed for In "Attach with Azure AD" dialog, "Tenant" drop down. Hence closing the bug.
PFA for reference
[2299_Closed.zip](https://github.com/microsoft/AzureStorageExplorer/files/11664334/2299_Closed.zip)
